OneWelcomeExampleApp/AppIconSwitcher.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2022 OneWelcome. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+class AppIconSwitcher {
```

Comment:
I guess now the whole class is not needed and you can move the logic to the interactor, right?

Reply:
I was thinking about it but AppIconSwticher is a bit more generic. I can do that if you want.